src/main/webapp/resources/js/pages/UserGroupsPage/index.js

```diff
@@ -30,10 +32,10 @@ export function UserGroups() {
         </div>
       }
     >
-      <Router style={{ height: "100%" }}>
-        <UserGroupsPage path="/groups" />
-        <UserGroupsDetailsPage path="/groups/:id" />
-      </Router>
+      <RolesProvider rolesFn={getUserGroupRoles}><Router style={{height: "100%"}}>
+        <UserGroupsPage path="/groups"/>
```

Comment:
Shouldn't we use setBaseUrl here? Probably best to set it at the top of the function as it used a couple of times

Reply:
This was already addressed in a different merge and will be addressed by merging in dev